cli/cmd/dashboard.go

```diff
@@ -20,19 +21,25 @@ const (
 
 	// showURL displays dashboard URLs without opening a browser.
 	showURL = "url"
+
+	// webDeployment is the name of the web UI's deployment in the install config
+	webDeployment = "linkerd-web"
+
+	// webPort is the port on which the web UI is served in the web pod
+	webPort = 8084
```

Comment:
mind adding a note that `webDeployment` and `webPort` must match values in `template.go`? perhaps eventually these should be shared constants.